### DISCUSSION TITLE
I’m geo-fenced in! Help!

### DISCUSSION THREAD

**Discussion by katblackk** [community/community#121256] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I need to know about GitHub projects in the Sacramento, CA, USA realm.  My device, an iPhone SE 3rd generation, seems to be a static target within a geo-targeted perameter, using a network dba T-mobile, with feature flags turned on, but no developer options for my use, with a managed “T-mobile” account, and a VPN, to which I am already signed in without me having known about it. 

I believe that the manager/developer/developers are using GitHub.

Katherine blackwell

**Comment by ebndev** [comment]
Hi @katblackk,

Although it sounds like your issue is with developers that have their project hosted on GitHub, GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the help that you need, but this space may not be the right place to support you with your issue.

If you believe this is something you'd like to report to GitHub, the best route to get this to the proper team is to use our [abuse reporting tools](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam). Here's all the info:

- [Reporting abuse or spam](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam)
- [Reporting a user](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-user)
- [Reporting an issue or pull request](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-an-issue-or-pull-request)
- [Reporting a comment](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-comment)

You can report behavior and content that violates community guidelines and terms. We are going to close this post, but for this and any future incidents, please refer to the links above.

Best of luck!
